package.json

```diff
@@ -23,5 +23,8 @@
   "devDependencies": {
     "dtslint": "github:Microsoft/dtslint#production",
     "types-publisher": "Microsoft/types-publisher#production"
+  },
+  "dependencies": {
+    "ngsijs": "^1.0.3"
```

Comment:
The file shouldn't be changed here, you can see the PR for reference: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22998/files